src/components/Navigation/Navbar.js

```diff
@@ -1,75 +1,78 @@
 import React, { useState } from 'react';
-import { Link } from 'react-router-dom';
+import { NavLink } from 'react-router-dom';
 import './Navbar.css';
+import logo from '../../assets/images/F2P-logo.svg';
 
-function Navbar() {
+export default function Navbar() {
   const [click, setClick] = useState(false);
   const handleClick = () => setClick(!click);
   const closeMobileMenu = () => setClick(false);
   return (
     <>
       <nav className="navbar">
         <div className="navbar-container">
-          <Link to="/" className="navbar-logo" onClick={closeMobileMenu}>
-            Farm2People
-            <i className="fas fa-tractor" />
-          </Link>
+          <NavLink to="/" onClick={closeMobileMenu}>
+            <img src={logo} alt="Logo" className="navbar-logo" />
+          </NavLink>
           {/* eslint-disable-next-line */}
           <div className="menu-icon" onClick={handleClick}>
             <i className={click ? 'fas fa-times' : 'fas fa-bars'} />
```

Comment:
I think that the width where the burger shows or the padding of the labels should be adjusted, there's a good amount of screen widths where the icons or links get cut off before it switches.

Reply:
Also, when it switches the logo becomes centered rather than at the leading edge of the bar. Is this on purpose?

Reply:
Great idea, I just changed the minWidth for the mediaQuery to be 1200, and b/c of that I think the centering of the logo looks a lot better now as well